app.py

```diff
@@ -1,12 +1,33 @@
 from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+from session_items import get_items, get_item, add_item, save_item
```

Comment:
I'd probably keep this as session given that we've got some similar app method names floating around here